app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +349,45 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  parseSpeciesMetricDecimalPlaces = species => {
```

Comment:
Lets call it `adjustMetricPrecision` ?
If this is only for species what happens with genus (i do not remember the full data structure...)?

Reply:
a sorted genus just contains a genusTaxId and a children species just contains speciesTaxId. they're both similar in structure:
`sorted genus  (32) [570, 561, 10509, 547, 255204, 92793, 1386, 1827195, 63265, 1743, 590, 59732, 390250, 1705353, 71927, -200541000, 1649459, 5324, 1263, 544, 110932, 1822464, 1301, 265, 1298, 80865, 1912216, 2053, 32008, 469, 48736, 1827]`